Makefile

```diff
@@ -17,6 +17,10 @@ package:
 forwarder:
 	$(MAKE) -C images forwarder
 
+.PHONY: hook
```

Comment:
Do we actually need any of these image targets split at this level? or can we just have a single `make images` target to call into `images/Makefile`.

Not important either way.